### PR TITLE
Fluffy homepage

### PR DESCRIPTION
**Pull request by Fluffy9**

### CODE REVIEW

**Comment by MSevey** [comment]
Pending Devops PR containing deploy to skynet action and prettier code formatter.

File: app/src/views/Home.vue [hunk 1]
```diff
@@ -1,18 +1,247 @@
...
-  name: 'Home',
+  name: "Home",
   components: {
-    HelloWorld
-  }
-}
+    BIcon,
+    SidebarMenu,
+  },
+  data() {
```
Comment (MSevey) [inline app/src/views/Home.vue:140]: What is this section containing? Should have a comment

Comment (MSevey) [reply app/src/views/Home.vue:140]: is this just filler stuff since nothing is hooked up to MySky yet?

File: app/src/views/Home.vue [hunk 2]
```diff
@@ -1,18 +1,247 @@
...
+        result = [...new Set(result)];
+        return result;
+      }
+    },
+  },
+  methods: {
+    /**
+     * Nice formatting for human readable dates
+     */
+    timeSince(date) {
```
Comment (MSevey) [inline app/src/views/Home.vue:206]: This is probably something that multiple places will use, so worth refactoring into a `utils` file that multiple locations can import.

**Review by MSevey — CHANGES_REQUESTED** [review]
Two minor changes. Then can we change this to just highlight the icon instead of having an extended bar?

Screen Shot 2021-09-15 at 9 13 34 AM

File: app/src/views/Home.vue (hunk 1 above)
Comment (Fluffy9) [reply app/src/views/Home.vue:140]: It's part of the Vue template. Every page has a data function which stores data related to that page

File: app/src/views/Home.vue (hunk 2 above)
Comment (Fluffy9) [reply app/src/views/Home.vue:206]: That was going to be my next big pull request. Refactoring to vuex store file and removing fake data

**Pull request closed without merging**